### PR TITLE
Looking for qunit instead of qunitjs

### PR DESCRIPTION
**Pull request by Johann-S**
it's seems karma-qunit look for `qunitjs` but it should look for `qunit`

### CODE REVIEW

**Comment by Johann-S** [comment]
I signed it!

File: lib/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function createPattern (pattern) {
 
 function initQUnit (files, qunit) {
   files.unshift(createPattern(path.join(__dirname, 'adapter.js')))
-  files.unshift(createPattern(require.resolve('qunitjs')))
+  files.unshift(createPattern(require.resolve('qunit')))
   if (qunit && qunit.showUI) {
     files.unshift(createPattern(require.resolve('qunitjs/qunit/qunit.css')))
```
Comment (dignifiedquire) [inline lib/index.js]: this needs to change as well probably

Comment (Johann-S) [reply lib/index.js]: Good catch 👍

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Comment by Johann-S** [comment]
you're welcome 😉

**Comment by dignifiedquire** [comment]
ugh, could you look into that and PR a test that runs a simple test using quint like we have in karma-mocha please?

**Comment by Johann-S** [comment]
I'll try to look at that later today

**Pull request merged**